src/redux/cofigureStore.js

```diff
@@ -0,0 +1,36 @@
+import { createStore } from 'redux';
+
+const initialState = {
+  books: [
+    {
+      id: 0,
+      categorie: 'Action',
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+    },
+    {
+      id: 1,
+      categorie: 'Science Fiction',
+      title: 'Dune',
+      author: 'Frank Herbert',
+    },
+    {
+      id: 2,
+      categorie: 'Economy',
+      title: 'Capital in the Twenty-First Century',
```

Comment:
- [ ] This should be in the `book/component` not in the `redux/configstore.js`, I think this is just an involuntary mistake.

![iiiiii](https://user-images.githubusercontent.com/61562626/139879251-012cc919-8383-428c-99ed-e307766119b3.jpeg)

Reply:
Hello, if you look closely I am not rendering nothing in the configureStore.js, the render happens in the components.

Reply:
- [ ] What my colleague was trying to say is that your list of books should be inside the `BookList` (example name) state, instead of a separate array outside the components, and your `BookList` component should pass each of these books as props for the `Book` component, kindly implement this changes before requesting another review 😄